megalista_dataflow/uploaders/google_ads/conversions/google_ads_ssi_uploader.py

```diff
@@ -83,108 +110,153 @@ def _do_upload(self, execution, offline_user_data_job_service, customer_id, curr
 
         # 1. Creates Job
         job_creation_payload = {
-            'type_': 'STORE_SALES_UPLOAD_FIRST_PARTY',
-            'external_id':  int(datetime.datetime.now().timestamp()*10e3),
-            'store_sales_metadata': {
-                'loyalty_fraction': 1.0,
-                'transaction_upload_fraction': 1.0
-            }
+            "type_": "STORE_SALES_UPLOAD_FIRST_PARTY",
+            "external_id": int(datetime.datetime.now().timestamp() * 10e3),
```

Comment:
Hi Pedro,
Is the external_id param used now? I remember that this was a parameter that was added but never used.
Thanks!

Reply:
it's still optional, but the documentation recommends it.